parallel-install/README.md

```diff
@@ -47,3 +47,52 @@ Once you have a configured `Deployment` instance, use the following functions ac
+| dstDir    | `string` | `myWorkspace/files`                                                                   | Path to which the file will be downloaded.                                                                                        |
+  
+`download.GetFiles` downloads a list of files and returns the paths to the downloaded files. Destination directory will be created if it does not exist.  
+Required parameters:
+| Parameter | Type       | Example value                                                                                                                                                                            | Description                                                                                                                           |
+| --------- | ---------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| files     | `[]string` | `[]string{"https://storage.googleapis.com/kyma-mps-dev-artifacts/prometheus-config-updater.zip", "https://storage.googleapis.com/kyma-mps-dev-artifacts/avs-bridge-noparent-1.3.5.tgz"}` | A list of URLs or local paths. For each URL, file will be downloaded. For each local path, file will be checked if it exists locally. |
+| dstDir    | `string`   | `myWorkspace/files`                                                                                                                                                                      | Path to which the files will be downloaded.                                                                                           |
+
+### Archive Package
+`archive` package allows you to decompress `zip` and `tar.gz`/`tgz` files.
+
+`archive.Unzip` decompress a `zip` file, moving all the files and folders within the zip file to an output directory.  
+Required parameters:
+| Parameter | Type     | Example value                                     | Description                                                                            |
+| --------- | -------- | ------------------------------------------------- | -------------------------------------------------------------------------------------- |
+| src       | `string` | `myWorkspace/files/prometheus-config-updater.zip` | Path to the zip file.                                                                  |
+| dst       | `string` | `myWorkspace/files/prometheusConfigUpdater`       | Path to the output directory in which files/folders within the zip file will be moved. |
+
+`archive.Untar` decompress a `tar.gz`/`tgz` file, moving all the files and folders within the `tar.gz`/`tgz` file to an output directory.  
```

Comment:
```suggestion
`archive.Untar` decompresses a `tar.gz`/`tgz` file, moving all the files and folders within the `tar.gz`/`tgz` file to an output directory.  
```

Reply:
I will change this also to untars instead of decompresses to be consistent with your other suggestion for having unzips instead of decompresses. Okay?

Reply:
I will change this also to untars instead of decompresses to be consistent with your other suggestion for having unzips instead of decompresses. Okay?

Reply:
I agreed with Nina to use "extracts"